go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/future-architect/vuls
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
Is this version specification necessary?

Reply:
Removed